src/Module/NodeInfo.php

```diff
@@ -48,7 +49,7 @@ public static function init()
 		}
 
 		if (($app->argc != 2) || ($app->argv[1] != '1.0')) {
-			System::httpExit(404);
+			self::printWellKnown($app);
```

Comment:
Why had you done this change? The 404 is thrown whenever the parameter count is correct or when something else than version "1.0" is requested.

Reply:
The good thing about the new routes is that we don't need to check for routes which aren't possible. So you don't have to check if there are too many or too less arguments, because in this case, the router wouldn't route to this class :-)

But I changed the behavior a little bit to make it more clear for the init